src/mlpack/methods/hoeffding_trees/hoeffding_tree_main.cpp

```diff
@@ -56,10 +56,10 @@ PARAM_STRING("output_model_file", "File to save trained tree to.", "M", "");
 
 PARAM_STRING("test_file", "File of testing data.", "T", "");
 PARAM_STRING("test_labels_file", "Labels of test data.", "L", "");
-PARAM_STRING("predictions_file", "File to output label predictions for test "
-    "data into.", "p", "");
-PARAM_STRING("probabilities_file", "In addition to predicting labels, provide "
-    "prediction probabilities in this file.", "P", "");
+PARAM_STRING("output_predictions_file", "File to output label predictions for"
+    "test data into.", "p", "");
+PARAM_STRING("output_probabilities_file", "In addition to predicting labels, "
+    "provide prediction probabilities in this file.", "P", "");
```

Comment:
Do you think that changing this parameter name violates our versioning policy for minor release changes?  I think I consider this a "public" part of the program, so I don't think we should change it until the next major release (3.0.0, I don't know when that will be), but I could probably be convinced either way.